### PR TITLE
Visual Fixes

### LINKED ISSUE #1447: Compliance Report | Left align Status column on main compliance reporting page |1|
**Task:**
Left align the status text in the Status column on the main compliance reporting page. 

- See attached screenshot

https://trello.com/c/iKP91oxi/1981-compliance-report-left-align-status-column-on-main-compliance-reporting-page-1

### PR DESCRIPTION
**Pull request by amichard**
#1447 #1448 #1449 #1451

Changelog:
- Changed the wording in some of the validation messages
- Left aligned status for Compliance Report
- Fixed null values for the address autocomplete in Exclusion Report
- Fixed the schedule totals blocking some parts of the spreadsheet
- Fixed vanishing buttons in the Exclusion Report

### CODE REVIEW

**Comment by amichard** [comment]
[Compliance Report | Left align Status column on main compliance reporting page |1|](https://trello.com/c/iKP91oxi/1981-compliance-report-left-align-status-column-on-main-compliance-reporting-page-1)

**Comment by amichard** [comment]
[Validation | Revise validation messaging |1|](https://trello.com/c/QcjbFmAf/1964-validation-revise-validation-messaging-1)

**Comment by amichard** [comment]
[Exclusion Reports | Null values showing in the address of transaction partner |1|](https://trello.com/c/dCNm3AOA/1968-exclusion-reports-null-values-showing-in-the-address-of-transaction-partner-1)

**Comment by amichard** [comment]
[Compliance Report | Floating windows block report data when viewing a submitted report |1|](https://trello.com/c/0jwsnmt0/1969-compliance-report-floating-windows-block-report-data-when-viewing-a-submitted-report-1)

File: backend/api/serializers/ExclusionReport.py [hunk 1]
```diff
@@ -119,16 +119,27 @@ class ExclusionReportUpdateSerializer(serializers.ModelSerializer):
     """
     Update Serializer for the Compliance Report
     """
-    status = ComplianceReportWorkflowStateSerializer(required=False)
-    type = SlugRelatedField(slug_field='the_type', read_only=True)
+    actions = serializers.SerializerMethodField()
```
Comment (plasticviking) [inline backend/api/serializers/ExclusionReport.py]: Do you think there's any need/desire to have `actor` here, too, as with compliance reporting?

Comment (amichard) [reply backend/api/serializers/ExclusionReport.py]: we can sure

**Pull request merged**